docs/reference/tab-widgets/troubleshooting/disk/decrease-data-node-disk-usage.asciidoc

```diff
@@ -0,0 +1,141 @@
+            "watermark": {
+              "high": "90%",
+              "high.max_headroom": "150GB"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+----
+// TEST[skip:illustration purposes only]
++
+The above means that in order to resolve the disk shortage we need to either drop our disk usage below the 90% or have
+more than 150GB available, read more on how this threshold works <<cluster-routing-watermark-high, here>>.
+
+. The next step is to find out the current disk usage, this will indicate how much space should be freed. For simplicity,
+our example has one node, but you can apply the same for every node over the relevant threshold.
++
+[source,console]
+----
+GET _cat/allocation?v&s=disk.avail&h=node,disk.percent,disk.avail,disk.total,disk.used,disk.indices,shards
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+node                disk.percent disk.avail disk.total disk.used disk.indices shards
+instance-0000000000           91     4.6gb       35gb    31.1gb       29.9gb    111
+----
+// TEST[skip:illustration purposes only]
+
+. The high watermark configuration indicates that the disk usage needs to drop below 90%. Consider allowing some
+padding, so the node will not go over the threshold in the near future. In this example, let's release approximately 7GB.
+
+. The next step is to list all the indices and choose which replicas to reduce.
++
+NOTE: The following command orders the indices with descending number of replicas and primary store size. We do this to
+help you choose which replicas to reduce under the assumption that the more replicas you have the smaller the risk if
+you remove a copy and the bigger the replica the more space will be released. This does not take into consideration any
+functional requirements, so please see it as a mere suggestion.
++
+[source,console]
+----
+GET _cat/indices?v&s=rep:desc,pri.store.size:desc&h=health,index,pri,rep,store.size,pri.store.size
+----
++
+The response will look like:
++
+[source,console-result]
+----
+health index                                                      pri rep store.size pri.store.size
+green  my_index                                                     2   3      9.9gb          3.3gb
+green  my_other_index                                               2   3      1.8gb        470.3mb
+green  search-products                                              2   3    278.5kb         69.6kb
+green  logs-000001                                                  1   0      7.7gb          7.7gb
+----
+// TEST[skip:illustration purposes only]
++
+. In the list above we see that if we reduce the replicas to 1 of the indices `my_index` and  `my_other_index` we will
+release the required disk space. It is not necessary to reduce the replicas of `search-products` and `logs-000001` does
+not have any replicas anyway. Reduce the replicas of one or more indices with the <<indices-update-settings,
+index update settings API>>:
```

Comment:
Shall we add a big warning here that the data redundancy and search throughput can be affected by this operation?

Reply:
yeap